### PR TITLE
add post on interpreting output

### PR DESCRIPTION
**Pull request by gregsdennis**
I received a question about this (again), and thought it'd make for a good post.

Also, I'm updating all of my links to my linkedin profile since I'm not really on twitter much anymore.

I'll probably cross-post on my blog as well.

### CODE REVIEW

**Comment by gregsdennis** [comment]
Not sure why the previews are giving 404...

**Comment by gregsdennis** [comment]
Rebased on `main` ☝️

File: pages/blog/posts/interpreting-output.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    link: https://www.linkedin.com/in/gregdennis/
+    byline: JSON Tooling Implementer, Specification & Tooling Architect @ Postman
+excerpt: How can successful validation output contain failure nodes?
+---
+
+I've received a lot of questions (and purported bugs) and had quite a few discussions over the past few years regarding JSON Schema output, and by far the most common is, "How can successful validation output contain failure nodes?"
+
+Let's dig in.
+
+_The last time we talked about output, it was to announce changes from the 2019-09/2020-12 version.  I'm going to use the new formats because it's easier to read and more compact._
```
Comment (jdesrosiers) [inline pages/blog/posts/interpreting-output.md:21]: Might be useful to link to that post.

**Review by jdesrosiers — APPROVED** [review]
lgtm

File: pages/blog/posts/interpreting-output.md (hunk 1 above)
Comment (gregsdennis) [reply pages/blog/posts/interpreting-output.md]: I had intended to, then I didn't.

**Pull request merged**